base/server/src/com/netscape/cms/servlet/csadmin/LDAPConfigurator.java

```diff
@@ -95,8 +95,11 @@ public void enableUSN() throws Exception {
     }
 
     public void setupSchema() throws Exception {
-        logger.info("Setting up schema");
+        logger.info("Setting up PKI schema");
         importSchemaFile("/usr/share/pki/server/conf/schema.ldif");
+
+        logger.info("Setting up ACME schema");
+        importSchemaFile("/usr/share/pki/acme/conf/database/ldap/schema.ldif");
```

Comment:
This is good.

As you have noted before in previous files, the ACME schema was defined twice:

- as a series of LDIF modify objects, in `...server/conf/schema.ldif` originally, and as of this PR in `...acme/conf/database/ldap/schema.ldif`.
- as an incomplete entry `cn=schema` entry in `...server/conf/schema-acme.ldif`.

I'll check whether the IPA schema update routines can handle LDIF modify objects, in which case we can delete the "plain object" definition.  Alternatively, we could modify `importSchemaFile` to interpret plain objects into modify operations, and remove the "modify" defintion (and move the "plain object" def to `...acme/conf/database/ldap/schema`).

Nothing to action here in this PR - just putting down my thoughts.

Reply:
Confirmed, the IPA schema update machinery handles this file just fine.  I will make a separate PR later to remove the duplicate definition.